app/process/model/bridge/servicefee/servicefee.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package servicefee
+
+import (
+	"math/big"
+	"sync"
+)
+
+type Servicefee struct {
+	serviceFee big.Int
```

Comment:
I think that it would be better if we store the service fee in `float` and set the actually service fee in percentage. F.e the smart contract has 5000 set as a service fee and PRECISION `100000`. That would mean that we will store:
`5000/100000=0.05` here as variable. 

Reasoning:
The fee calculator should not know the PRECISION of the smart contract, even worse -> have it hardcoded in its code. Ideally the service fee should store the exact fee percentage

Reply:
Will be fixed in another PR